src/styles/blocks/header.scss

```diff
@@ -0,0 +1,146 @@
+.header {
+  position: relative;
+  padding-bottom: $headerSliderPadding;
+  background-color: $c_dark-grey ;
+
+  @include screen-md() {
+    padding-bottom: 205px;
+    border-bottom-right-radius: 30px;
+    border-bottom-left-radius: 30px;
+    overflow: hidden;
+  }
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 28px;
+    padding: 20px 0;
+
+    @include screen-sm() {
+     margin-bottom: 36px;
+    }
+
+    @include screen-md() {
+     margin-bottom: 104px;
+     padding: 34px 0;
+    }
+  }
+
+  &__nav {
+    display: none;
+
+    @include screen-md() {
+      display: flex;
+      gap: 48px;
+    }
+  }
+
+  &__menu-opener {
+    margin-left: 36px;
+    width: 18px;
+    height: 8px;
+
+    background-image: url(../icons/menu.svg);
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+
+    @include screen-md() {
```

Comment:
i can see that in sometimes you add parentheses and sometimes you don't. be consistent
(you do not really need them here, because no parameters are being passed)